axum-handle-error-extract/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "axum-handle-error-extract"
+version = "0.1.0"
+authors = ["David Pedersen <david.pdrsn@gmail.com>"]
+categories = ["asynchronous", "network-programming", "web-programming"]
+description = "Error handling layer for axum that supports extractors and async functions"
+edition = "2018"
+homepage = "https://github.com/tokio-rs/axum"
+keywords = ["http", "web", "framework"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/tokio-rs/axum"
+
+[dependencies]
+axum = { path = "../axum" }
+tower-service = "0.3"
+tower-layer = "0.3"
+tower = { version = "0.4", features = ["util"] }
+pin-project-lite = "0.2"
```

Comment:
If the dependencies are meant to be sorted alphabetically, this one should move up to after axum 😉

If you want to automate sorting / check it in CI here and in other `Cargo.toml`s, there is [cargo-sort](https://github.com/DevinR528/cargo-sort).

Reply:
Hehe I don't care too much about keeping those sorted.